src/ring/middleware/oauth2.clj

```diff
@@ -38,10 +38,19 @@
   (= (get-in request [:session ::state])
      (get-in request [:query-params "state"])))
 
+(defn- coerce-to-int
+  [n]
```

Comment:
The argument list probably doesn't need to be on it's own line. `(defn- coerce-to-int [n]` is just as clear.